apps/builder/app/builder/features/style-panel/property-label.tsx

```diff
@@ -320,13 +320,15 @@ export const PropertySectionLabel = ({
  */
 export const PropertyInlineLabel = ({
   label,
+  title,
   description,
   properties,
   disabled,
 }: {
   label: string;
+  title?: string;
```

Comment:
what this for?

Reply:
<img width="351" alt="Screenshot 2024-09-03 at 10 47 33 AM" src="https://github.com/user-attachments/assets/18e8877c-aa0a-4eb4-97fe-73bf8a1757a4">
<img width="353" alt="Screenshot 2024-09-03 at 10 47 19 AM" src="https://github.com/user-attachments/assets/62705a94-179b-410a-8670-9b3b51be9e8a">

We changed the design here to hide the syntaxes when it not actually a valid css property. So, instead we show it as label. So, we have a label and a title. Not necessarily be the same so.

Reply:
we don't want to show box-shadow-offset-x because its not css standard, so we need something that is more descriptive in the title than what we have in the label, because X is not going to be clear, Offset X is clear

Reply:
TBH in case of Offset X and Offset Y we could say Offset X and Offset Y

Reply:
But we need to make it consistent everywhere, not sure how about other properties